adb_graphics/default_specs.yml

```diff
@@ -1301,6 +1301,24 @@ trc1:
     title: Vertically Integrated Smoke
     transform: conversions.to_micro
     unit: $mg/m^2$
+  lvl5:
+    clevs: [1, 2, 4, 6, 8, 12, 16, 20, 25, 30, 40, 60, 100, 200]
+    colors: smoke_colors
+    ncl_name: MASSDEN_P0_L105_{grid}
+    ticks: 0
+    title: 1000ft AGL Smoke
+    vertical_index: 4
+    wind: True
+    unit: $\mu g/m^3$
+  lvl12:
+    clevs: [1, 2, 4, 6, 8, 12, 16, 20, 25, 30, 40, 60, 100, 200]
```

Comment:
Please consider using an anchor and reference to reduce duplication of information here.

Reply:
I was struggling with anchors before, and planned to revisit it once I got things straightened out.  The next commit will have that.